src/state/index.ts

```diff
@@ -5,18 +5,18 @@ import { composeWithDevTools } from 'redux-devtools-extension';
 import { mapStringToEnvironment } from '../common/mappers/environment.mapper';
 import { Environment } from '../common/types/environment.type';
 import { counterReducer, CounterState } from './counter/counter.reducer';
-import { homepageReducer, HomepageState } from './homepage/homepage.reducer';
 import { DEFAULT_STATE } from './state.constants';
+import { surveysReducer, SurveysState } from './surveys/surveys.reducer';
 
 export interface AppState {
     counterState: CounterState;
-    homepageState: HomepageState;
+    surveys: SurveysState;
     router: RouterState;
 }
 
 export const reducers = (history: History<any>): { [key in keyof AppState]: Reducer<any, any> } => ({
     counterState: counterReducer,
-    homepageState: homepageReducer,
+    surveys: surveysReducer,
```

Comment:
Wdyt about changing the naming here? So `surveys` becomes `surveysState`, `router` becomes `routerState`?

I far prefer accessing `surveysState.surveys` over `surveys.list`

Reply:
Yes please.

Reply:
Apparently the router state must be named `router` when using connected-react-router. Otherwise it throws the following error: `Error: Could not find router reducer in state tree, it must be mounted under "router"`.